resources/styles/components/performance/index.less

```diff
@@ -25,6 +25,17 @@
     max-height: 100%;
   }
 
+  // removes the display as table, table-row, & table-cell
+  // from the wrappers so the header-cell below can set
+  // it's width to 100% with text-overflow: ellipsis;
```

Comment:
Hrm?  I'm guessing that the ' in "it's" was throwing your syntax highlighting off?  I've removed it and reworded the comment to be a bit more concise.